### ISSUE TITLE
t1-linear crash files cannot be written

### ISSUE THREAD

**Issue by 14thibea** [aramis-lab/clinicadl#123]
Hi @mdiazmel,

There is a small bug in `t1-linear` pipeline when launched with ClinicaDL (it works with Clinica).
This is due to the last line of `t1_linear_cli.py`:
```
print_crash_files_and_exit(args.logname, pipeline.base_dir)
```

There is no `logname` in `args` in the case of ClinicaDL as this argument is added at a higher level in Clinica in `cmdline.py`:
```
    parser.add_argument("-l", "--logname",
                        dest='logname',
                        default="clinica.log",
                        metavar=('file.log'),
                        help='Define the log file name (default: clinica.log)')
```

This argument (and verbose) should be added to the general command line of `clinicadl preprocessing run`.

**Comment by 14thibea** [comment]
It seems there is also a bug with `clinicadl preprocessing extract-tensor` while the corresponding pipeline in clinica is working normally.

**Comment by mdiazmel** [comment]
Closed by PR #126
